content/syllabuses/data-sci.md

```diff
@@ -11,7 +11,7 @@ _db_id: 2
 - {{% contentlink path="agile/agile-scrum" %}}
 - NCIT 18: Agile project management
 - NCIT 19: Agile meetings
-- {{% contentlink path="workshops/how-to-be-a-professional" %}}
+- {{% contentlink path="topics/how-to-be-a-professional" %}}
```

Comment:
Can this be changed to a topic that covers the relevent info instead of an outdated presentation?